dragonfly/implementations/daq_run_interface.py

```diff
@@ -128,7 +142,7 @@ def _do_postrun_gets(self):
             self._run_snapshot['LOGS'].update(these_snaps)
 
     def determine_RF_ROI(self):
-        raise core.exceptions.DriplineMethodNotSupportedError('subclass must implement RF ROI determination')
+        logger.warning('subclass must implement RF ROI determination; skipped')
```

Comment:
Not sure the ROI determination is optional, saving RSA data without knowing/saving where we are looking at it bad. A RaiseException prevents us from not saving them.